core/model/src/main/java/org/eclipse/sw360/antenna/model/license/LicenseInformation.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) Bosch.IO GmbH 2020.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.model.license;
+
+import java.util.List;
+
+public interface LicenseInformation {
+    String evaluate();
+    String evaluateLong();
```

Comment:
Here we have the chance to drop the long. I think this should be renamed to evaluateUseNames. Is this method really needed? I would expect that evaluate should be enough, especially since SPDX-Identifier are quite unterstandable.